bin/skel.py

```diff
@@ -337,7 +339,9 @@ def problem_source_dir(problem: Problem):
     contest_yml = (config.TOOLS_ROOT / "skel/gitlab_ci/contest.yaml").read_text()
     contest_path = Path(".").resolve().relative_to(git_root_path)
     changes = "".join(
-        "      - " + str(problem_source_dir(problem)) + "/problem_statement/**/*\n"
+        f"      - {problem_source_dir(problem)}/{latex.PdfType.PROBLEM.path().parent}/**/*\n"
+        f"      - {problem_source_dir(problem)}/{latex.PdfType.PROBLEM_SLIDE.path().parent}/**/*\n"
+        f"      - {problem_source_dir(problem)}/{latex.PdfType.SOLUTION.path().parent}/**/*\n"
         for problem in problems
     )
```

Comment:
I think something like this should work :smile:
```suggestion
    changes = "".join(
        f"      - {problem_source_dir(problem)}/{pdf_type.path().parent}/**/*\n"
        for problem in problems
        for pdf_type in latex.PdfType
    )
```